app/static/js/event_list.js

```diff
@@ -5,8 +5,30 @@ $(document).ready(function(){
     $("#rsvpBtn").click(function(){
         rsvpForEvent($("#rsvpBtn").val())
     })
+    $("#toggleButton").on("click", toggleButton)
+    localStorage.removeItem("toggleState");
```

Comment:
If you remove this when the page loads, how do you use it to decide whether to check the checkbox or not?